packages/cli/src/bin/ceramic.ts

```diff
@@ -24,6 +24,8 @@ program
     .option('--max-healthy-memory <decimal>', 'Fraction of total memory usage considered healthy. Defaults to 0.7')
     .option('--cors-allowed-origins <list>', 'Space-separated list of strings and/or regex expressions to set for Access-Control-Allow-Origin . Defaults to all: "*"')
     .option('--disable-anchors', 'Allows Ceramic Daemon to start up without a configured anchor service. Any anchor requests made will fail.')
+    .option('--no-restore-streams', 'Disable loading pinned stream state into memory at startup')
+    .option('--sync <string>', 'Global modes for syncing streams. One of: "prefer-cache", "sync-always", or "never-sync". Defaults to "prefer-cache"')
```

Comment:
Can we call this `--sync-override`?  Or something else to clarify that it overrides the behavior of whatever the client requests?  Would also be good to clarify that in the flag description.